content/blog/2021/outreach-partners.md

```diff
@@ -0,0 +1,39 @@
+Title: We are looking for outreach partners for PyCon India 2021
+Date: 2021-04-06 23:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2021
+Slug: outreach-partners
+Summary: We are looking for outreach partners that can help spread the word about our conference in communities all over the world and bring fresh faces to the platform in hopes of creating a more inclusive, accessible conference this year.
+
+[**PyCon India**](https://in.pycon.org/2021/) is looking for outreach partners that can help spread the word about our conference in communities all over the world and bring fresh faces to the platform in hopes of creating a more inclusive, accessible conference this year.
+
+PyCon India is the largest gathering of Python learners, programmers, and enthusiasts in India who come from all over the world once a year to celebrate the programming language and share their experiences, insight, and expertise with the community. This year we like to bring forward positive change, and make the conference even more inclusive by bringing forth combined efforts of our communities in helping PyCon India reach more viewers.
+
+## What we expect from you
+
+We would like to invite your community to be an outreach partner for PyCon India 2021. You can help us spread the word about the conference among your members who are looking to learn more regarding Python. We expect our outreach partners to help us in the following ways:
+
+* Introduce your community to PyCon India 2021 by creating an event on your platform and share the event on your Social Media platforms.
+* Share the content from the conference with your community members on your mailing list, social media channels, and messaging groups.
+* Spread the word about our call for proposals for talks, workshops & dev sprint and help us find speakers, mentors, and reviewers from your community for the conference.
+* Update the PyCon India team after sharing the content from the conference on your platforms so that the team can keep track of our promotions.
+
+## What you can expect from us
+
+For supporting the conference, our outreach partners can expect the following in return from PyCon India:
+
+* We will Acknowledge your community in opening and closing address
+* We will display your community logo on the conference screen between sessions
+* We will display your Logos and Names on the conference website.
+* We will also promote your community on our social media channels as a supporter of the conference.
+
+## Disclaimer
+
+* There will not be any discounts on ticket prices or any other financial benefits of any kind, since PyCon India 2021 is a volunteer-driven event.  
+* Outreach partners for PyCon India 2021 will not be carried forward to further years.  
+
+Please be mindful of others and respect each other and be clear in your communication while addressing the audience at the conference/expo booths and follow the conference [Code of Conduct](https://in.pycon.org/2021/coc.html).
+
```

Comment:
```suggestion

The PyCon India team reserves the right to remove you as an outreach partner in case of any violation of the Code of Conduct, or in case you are found not sharing the conference updates with your community and taking advantage of the conference for your own promotion. 
```

I wanted a review on this before adding this to the blog.

Reply:
Maybe don't outright mention "remove". Mention something like, "In case of any violation of ..... PyCon India reserves the right to take necessary action as deemed fit and needed of the situation created."
something like that..

basically, it's a broader action panel now.

Reply:
>> and taking advantage of the conference for your own promotion. 
remove this.

Reply:
I've updated the suggestion. Please take a look again.

Reply:
Can we think of an alternate for this : "or if you are found not sharing the conference updates with your community" ?
It sounds very transactional. After-all, we want to bring people together like a community, not make business deal.

(Just my thoughts).

Reply:
```suggestion

The PyCon India team reserves the right to take the necessary action as deemed fit and needed in case of any violation of the Code of Conduct, or if you are found not abiding by the program guidelines.

```